src/main/typescript/components/form/parts/index.ts

```diff
@@ -33,4 +32,4 @@ export type DepositFormMetadata =
     & PrivacySensitiveDataFormData
     & DepositLicenseFormData
 
-export type DepositFormData = DataFormData & DepositFormMetadata
+export type DepositFormData = DepositFormMetadata
```

Comment:
Since the `DepositFormMetadata` is already exported, you can remove this line and redirect all it's uses to the former.